templates/skeleton/app/entry.server.tsx

```diff
@@ -46,6 +48,24 @@ export default async function handleRequest(
   responseHeaders.set('Content-Type', 'text/html');
   responseHeaders.set('Content-Security-Policy', header);
 
+  if (context.analyticsTokens) {
+    const serverTimingValue = createAnalyticsServerTimingHeader(context.analyticsTokens);
+    if (serverTimingValue) {
+      responseHeaders.set('Server-Timing', serverTimingValue);
+    }
+    
+    const cookieHeaders = createAnalyticsCookieHeaders(
+      context.analyticsTokens,
+      {
+        requestUrl: request.url,
+      },
+    );
+    
+    cookieHeaders.forEach(cookieHeader => {
+      responseHeaders.append('Set-Cookie', cookieHeader);
+    });
+  }
+
   return new Response(body, {
     headers: responseHeaders,
     status: responseStatusCode,
```

Comment:
This is creating multiple cookies (`cookieHeaders` is an array of 4-5 values, see [source](https://github.com/Shopify/hydrogen/pull/3144/files#diff-9eb4f8daaaad350566a7cc609bc3aacb1634d69eacfd2af778511425b2dd9b57R157)).

I thought we only wanted to have 1, so shouldn't we merge these values here into one (or at the source, probably)?